From 275c787c2c5909faee914520271042df36c26da3 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 8 May 2017 19:14:38 +0800
Subject: [PATCH 1759/5242] MLK-14695-4 ARM64: dts: freescale: imx8qm: add sc
 rtc node

commit  9ac86e07cc5b47b541d7191f9ee04026e31d4eb7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add system controller RTC support for i.MX8QM.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 7ddcb08..7097b1d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -742,6 +742,10 @@
 		};
 	};
 
+	rtc: rtc {
+		compatible = "fsl,imx-sc-rtc";
+	};
+
 	i2c0: i2c@5a800000 {
 		compatible = "fsl,imx8qm-lpi2c", "fsl,imx7ulp-lpi2c";
 		reg = <0x0 0x5a800000 0x0 0x4000>;
-- 
1.7.9.5

